core/server/master/src/main/java/alluxio/web/WebInterfaceConfigurationServlet.java

```diff
@@ -39,8 +40,8 @@
 public final class WebInterfaceConfigurationServlet extends HttpServlet {
   private static final long serialVersionUID = 2134205675393443914L;
   private static final String ALLUXIO_CONF_PREFIX = "alluxio";
-  private static final Set<String> ALLUXIO_CONF_EXCLUDES = new HashSet<>(
-      Arrays.asList(PropertyKey.MASTER_WHITELIST.toString()));
+ private static final Set<String> ALLUXIO_CONF_EXCLUDES = Sets.newHashSet(
```

Comment:
I think the indentation is off on this line - you can see if the style passes by running `mvn checkstyle:check` on your computer

Reply:
I have changed it, check it please!